src/repeat-prevent.coffee

```diff
@@ -21,8 +21,8 @@ module.exports = (robot) ->
 		# Tidy the old comments
 		for comment, timestamp of scopeConsidered when timestamp.isBefore(horizon)
 			delete scopeConsidered[comment]
-		# Allow the response if it's not in our memory
-		if (not scopeConsidered[response]?)
+		# Allow the response if bot name is mentioned or it's not in our memory
+		if (context.response.message.text.toLowerCase().includes(context.response.robot.name.toLowerCase())) or (not scopeConsidered[response]?)
```

Comment:
Nit: Robot name here should probably come out as a separate var, to make this a little more readable.